tests/integration/specs/multiple-resource-links-test.js

```diff
@@ -65,31 +79,33 @@ module('integration/specs/multiple-resource-links-test', {
 });
 
 asyncTest('GET /posts/1 calls later GET /posts/1/comments when Posts has async comments', function() {
-    var models = setModels({
-        authorAsync: true
-    });
-    env = setupStore(models);
+  var models = setModels({
+      authorAsync: true
+  });
+  env = setupStore(models);
 
-    fakeServer.get('/posts', responses.posts_not_compound);
-    fakeServer.get('/posts/1/author', responses.post_1_author);
-    fakeServer.get('/posts/2/author', responses.post_2_author);
+  fakeServer.get('/posts', responses.posts_not_compound);
+  fakeServer.get('/posts/1/author', responses.post_1_author);
+  fakeServer.get('/posts/2/author', responses.post_2_author);
 
-    Em.run(function() {
-        env.store.find('post').then(function(records) {
-            equal(records.get('length'), 2, 'there are 2 posts');
+  Em.run(function() {
+      env.store.find('post').then(function(records) {
+          equal(records.get('length'), 2, 'there are 2 posts');
 
-             var post1 = records.objectAt(0);
-             var post2 = records.objectAt(1);
-             var promises = [];
+           var post1 = records.objectAt(0);
+           var post2 = records.objectAt(1);
+           var promises = [];
 
-             promises.push(post1.get('author').then(function(author) {
-                equal(author.get('name'), 'dhh', 'post1 author');
-             }));
-             promises.push(post2.get('author').then(function(author) {
-                 equal(author.get('name'), 'ado', 'post2 author');
-             }));
+           promises.push(new Promise(function(resolve, reject) { resolve('asdf'); }));
```

Comment:
This fails when running `npm run test` because `Promise` is not found. Needs to be `Ember.RSVP.Promise` instead.